mattermost-plugin/server/configuration.go

```diff
@@ -71,21 +68,27 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 }
 
 // OnConfigurationChange is invoked when configuration changes may have been made.
-func (p *Plugin) OnConfigurationChange() error {
+func (p *Plugin) OnConfigurationChange() error { //nolint
```

Comment:
Not related changes here

Reply:
This will go away once the FeatureFlags PR gets merged to master and this PR updated to master. I had to merge in the FF PR in order to test.